content/en/monitors/monitor_types/log.md

```diff
@@ -27,9 +27,9 @@ As you define the search query, the graph above the search fields updates.
 
 * If you have [multiple log indexes][2], select the index to search.
 * Construct a search query using the same logic as a [log explorer search][3].
-* Choose a [facet][4] or [measure][5] to monitor.
-    * Facets display `Count`.
-    * Measures display `Measure` with an aggregation choice: `min`, `avg` (default), `sum`, `median`, `pc75`, `pc90`, `pc95`, `pc98`, `pc99`, or `max`.
+* Choose to monitor over a log count, [facet][4], or [measure][5]:
```

Comment:
Well that's not accurate anymore then.

Because if a measure is selected we do not monitor over a log count but over a numerical value that is in the log (if my max response time is above 2s for instance) which can be very very different from a monitor over a log count matching a query.

Reply:
Should we split this documentation in two parts:

1. Setup, what you can do?
2. Use cases, when you want to do it? (count, cardinality, measures)

Reply:
Sounds like this should be 3 bullet points then. I will update.

If you want a `Use cases` section, it can be added to the bottom above `Further Reading`. We would need guidance on that and I suggest doing it in a separate PR.  This PR is to clean-up existing content.